src/NUnitEngine/Addins/nunit.v2.driver.tests/AssertSyntaxTests.cs

```diff
@@ -4,825 +4,825 @@
 // obtain a copy of the license at http://nunit.org
 // ****************************************************************
 
-using System;
 using System.Collections;
+using NUnit.Framework;
 using NUnit.Framework.Constraints;
 
-namespace NUnit.Framework.Tests
+namespace NUnit.Engine.Drivers.Tests
```

Comment:
These are  actually the framework tests of the NUnit 2.6.4 framework, copied as-is into the tests for this driver.

No harm in the change however. Same thing goes for the formatting changes from NUnit 2.x to 3.0 standards.